corehq/apps/userreports/expressions/specs.py

```diff
@@ -2,7 +2,15 @@
 from couchdbkit.exceptions import ResourceNotFound
 from jsonobject import JsonObject, StringProperty, ListProperty, DictProperty
 from jsonobject.base_properties import DefaultProperty
-from corehq.apps.userreports.expressions.getters import DictGetter, NestedDictGetter
+from corehq.apps.userreports.expressions.getters import (
+    DictGetter,
+    NestedDictGetter,
+    TransformedGetter,
+)
+from corehq.apps.userreports.indicators.specs import (
+    DataTypeProperty,
+    _transform_from_datatype,
```

Comment:
I think this should move to `expressions.getters`. 

fyi, it's a python convention that generally starting a function with `_` means you don't expect it to be imported outside that module